src/main/java/org/prebid/server/spring/config/model/AccountConfigurationProperties.java

```diff
@@ -0,0 +1,58 @@
+package org.prebid.server.spring.config.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.settings.model.Account;
+import org.prebid.server.settings.model.AccountAnalyticsConfig;
+import org.prebid.server.settings.model.AccountGdprConfig;
+import org.prebid.server.settings.model.AccountStatus;
+import org.springframework.validation.annotation.Validated;
+
+@Validated
+@Data
+@NoArgsConstructor
+public class AccountConfigurationProperties {
+
+    private String priceGranularity;
+
+    private Integer bannerCacheTtl;
+
+    private Integer videoCacheTtl;
+
+    private Boolean eventsEnabled;
+
+    private Boolean enforceCcpa;
+
+    private String gdpr;
```

Comment:
Why cant we use AccountGdprConfig class

Reply:
Because we don't want to add `@Data @NoArgsConstructor` annotations to `AccountGdprConfig`, i.e. make it a Spring configuration properties POJO.

Reply:
Probably you mean `AccountAnalyticsConfig` bc AccountGdprConfig is already have this annotation.

Reply:
Good point. `AccountGdprConfig` shouldn't have had these annotations in the first place.

Reply:
Probably my mistake. I copy from GdprConfig which we get from yaml. 
But why we don't want to make this @Data ? Because of mutability ?

Reply:
Yes, it's a convention for value objects throughout PBS codebase to aspire for immutability unless it is impractical.

Another reason to take into consideration - using `AccountGdprConfig` to represent data in two different places (PBS configuration and account configuration in DB/file/etc) strongly couples these sources and forces changes in one place (account config for example) to affect another place (PBS config) and vice versa. Actually This is what Single Responsibility Principle is called to protect from.